### ISSUE TITLE
embed traits.build version in database `config/metadata.yml` file

### ISSUE THREAD

**Issue by ehwenk** [traitecoevo/traits.build#124]
Eventually we need to work out how to embed the version of traits.build used to build a database into the database's metadata.yml file - and maybe also the `build` object output.

**Comment by dfalster** [comment]
Hi @ehwenk 

Actually, looks like we already have it available in `build_info`, e.g.

```
austraits$build_info$session_info$otherPkgs$traits.build$Version
```

We could make it more prominent, eg. put at higher level

```
austraits$build_info$traits.build_version 
```

But I'm not sure untying more is needed than what we already have access to?

thoughts?

**Comment by ehwenk** [comment]
Ah perfect! I didn't know about that. Just thinking forward to referencing versions of `traits.build` as well as database versions.

**Comment by ehwenk** [comment]
closed with commit b5e9696
